docs/iris/src/common_links.inc

```diff
@@ -0,0 +1,24 @@
+.. _SciTools: https://github.com/SciTools
+.. _Iris: https://github.com/SciTools/iris
+.. _Iris GitHub: https://github.com/SciTools/iris
+.. _iris mailing list: https://groups.google.com/forum/#!forum/scitools-iris
```

Comment:
Not something for this PR, but given its continued inclusion here, I'd just like to raise that checking the Google Group needs to be more enshrined as part of the role of a core dev. It's not something I check nearly enough.

Reply:
I think the Google Groups usage is something we need to review - either start using it or make a statement of intent to stop using it.  I will capture this as an issue (discussion piece)

Reply:
Discussion issue raised about google groups usage, see https://github.com/SciTools/iris/issues/3860